packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -522,7 +522,11 @@ class DevFS {
           _scanFile(deviceUri, file);
       }
     } catch (e) {
```

Comment:
yes please. we should avoid ever catching all exceptions (unless we reraise them, or report them and quit, or something).